marklogic-data-hub/src/main/resources/hub-internal-config/security/privileges/data-hub-write-mapping.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Thoughts on using xdmp.securityAssert in the step endpoints? Would require conditionals on stepDefinitionType